src/webhooks/registry.ts

```diff
@@ -274,9 +269,8 @@ export function createRegisterAll(config: ConfigInterface) {
     const topics = getTopics();
 
     for (const topic of topics) {
-      const handler = getHandler(topic);
+      const handler = getHttpHandler(topic);
```

Comment:
Is there a particular reason for us to be exporting an interface in line 288 here?